.github/workflows/sca_scan.yaml

```diff
@@ -0,0 +1,62 @@
+name: Software Composition Analysis
+
+on:
+ # Runs when a pull request review is being submitted
+  pull_request_review:
+   types: [submitted]
+   branches:
+     - 'main'
+
+  # Run this workflow manually from Actions tab
+  workflow_dispatch:
+    inputs:
+      branch_to_test:
+        description: 'Branch or tag to run test'
+        required: true
+        default: 'main'
+        type: string
+
+# Allow one concurrent deployment
+concurrency:
+  group: ${{ github.repository }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  software-composition-analysis:
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+
+    - name: Checkout Code (Pull_Request_Review)
+      if: github.event_name == 'pull_request_review'
+      uses: actions/checkout@v3
+      with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          submodules: recursive
+
+    - name: Checkout Code (Workflow_Dispatch)
+      if: github.event_name == 'workflow_dispatch'
+      uses: actions/checkout@v3
+      with:
+          ref: ${{ inputs.branch_to_test }}
+
+    - name: Set Branch Variable (Pull_Request_Review)
+      if: github.event_name == 'pull_request_review'
+      run: |
+          echo "BRANCH=branch" >> "$GITHUB_ENV"
+
+    - name: Set Branch Variable (Workflow_Dispatch)
+      if: github.event_name == 'workflow_dispatch'
+      run: |
+          echo "BRANCH=${{ inputs.branch_to_test }}" >> "$GITHUB_ENV"
+
+    - name: Install & Scan MOONSHOT
+      run: |
+          echo $BRANCH
+          cd moonshot
+          npm install
```

Comment:
sorry but can check why you need to run npm install for moonshot?

Reply:
oh...my bad..will update it accordingly. thanks for pointing it out!

Reply:
Fix Python Installation